packages/ui/ui/slider.tsx

```
@@ -0,0 +1,41 @@
+ * @returns A styled range input element
+ *
+ * @example
+ * ```tsx
+ * <Slider min={1} max={5} value={3} step={1} onChange={handleChange} />
+ * ```
+ */
+function Slider({ className, ref, ...props }: SliderProps) {
+	return (
+		<input
+			type="range"
+			className={cn(
```

ESLint

[error] Props should be sorted alphabetically

(react/jsx-sort-props)